tests/ui/implicit_saturating_sub.rs

```diff
@@ -1,7 +1,22 @@
 // run-rustfix
-#![allow(unused_assignments, unused_mut, clippy::assign_op_pattern)]
+#![allow(unused_assignments, unused_variables, unused_mut, clippy::assign_op_pattern)]
```

Comment:
Is this really needed?

Reply:
Facing wierd error without it as `b` is declared but never used, but `b` is certainly in used.

Reply:
In this case, renaming  the variable to start with an underscore (e.g. `_b`) is the canonical way to get rid of the error.